internal/handlers/pg/cmd_query.go

```diff
@@ -51,10 +51,10 @@ func (h *Handler) CmdQuery(ctx context.Context, query *wire.OpQuery) (*wire.OpRe
 
 		default:
 			msg := fmt.Sprintf("CmdQuery: unhandled command %q", cmd)
-			return nil, common.NewErrorMsg(common.ErrNotImplemented, msg)
+			return nil, common.NewCommandErrorMsgWithArgument(common.ErrNotImplemented, msg, "query")
```

Comment:
Command is not an argument

Reply:
Yeah I noticed that so I'm not doing that mistake in newer changes and will replace commands that I've already placed